tests/snapshots/execution_spec__input-type-protected-error.md_errors.snap

```diff
@@ -0,0 +1,24 @@
+---
+source: tests/execution_spec.rs
+expression: errors
+---
+[
+  {
+    "message": "Input types can not be protected",
+    "trace": [
+      "Input",
+      "name",
+      "@protected"
```

Comment:
```suggestion
      "@protected"
```

Reply:
Issue created to address this: https://github.com/tailcallhq/tailcall/issues/1599